src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -240,6 +250,23 @@ private CategoryPill fillPostCategoryWith(final PostsCreator postsCreator, final
     return categoryPill;
   }
 
+  private String addLinkToDescription(final PostsCreator postsCreator, final String description) throws MalformedURLException {
```

Comment:
Do you think it's a good place for this method? I guess it fits more in `BasePostsCreator` class, or any other page object.